tap_fullstory/__init__.py

```diff
@@ -101,13 +100,13 @@ def request(endpoint, params=None):
         resp = SESSION.send(req)
         timer.tags[metrics.Tag.http_status_code] = resp.status_code
 
+    resp.raise_for_status()
```

Comment:
On requests with a 429 response, waiting to raise an exception for the 429 was causing other exceptions in the JSON code below.